### ISSUE TITLE
Include semicolon in regex to standardize having a space after it

### ISSUE THREAD

**Issue by dwhieb** [Nisinoon/Nisinoon#212]
https://github.com/dwhieb/Nisinoon/blob/45b67dfb36b4511d388cee8da3ed14066cf4f33b/data/utilities/parsePages.js#L1-L9

Also, why are you filtering out non-integers here?

You should probably not treat the "pages" as numbers at all. They should be plain text, so people can enter section numbers, etc.
